### ISSUE TITLE
No profiles in "Profile class" menu

### ISSUE THREAD

**Issue by OleMussmann** [ESDLMapEditorESSIM/ESSIMDocumentation#1]
Following the first tutorial at https://essim-documentation.readthedocs.io/en/latest/tutorials/index.html#tutorial-1-basic-energy-system I can find no profiles in the "Profile class". The bold entries are not selectable.

![esld_menu](https://user-images.githubusercontent.com/14004859/171646581-b37c5930-473b-462f-a836-b976994e0e39.png)

Do these have to be imported in a previous step or during the setup of the ESDL Map Editor?

Host system: Ubuntu 20.04
Deployment: Docker, from https://github.com/ESDLMapEditorESSIM/docker-toolsuite

**Comment by ewoudwerkman** [comment]
Did you do `Step 8. Upload some profiles` from  https://github.com/ESDLMapEditorESSIM/docker-toolsuite ? There are some example profiles in the `Data\Profiles\standard_profiles.csv` folder, you can upload in the ESDL MapEditor

**Comment by OleMussmann** [comment]
Thank you for your very quick reply @ewoudwerkman . I indeed missed that step, my apologies.

Are the `standard_profiles.csv` energy values or power/temperature values? Should they be summed or averaged?

![profile_menu](https://user-images.githubusercontent.com/14004859/171650804-e5626f95-3d1d-4e23-828b-4040c442688e.png)

Edit:
Further down in the tutorial I found the line
`Select ‘Energy in GJ’ from the dropdown list ‘Quantity and Unit’.`
so I assume it is energy values which need to be summed.
